### DISCUSSION TITLE
Noifications to telegram group instead of a channel – is it possible?

### DISCUSSION THREAD

**Discussion by Get-108** [umputun/remark42#1756] [General]
Is it possible to send notification by bot to telegram group, not a channel?

**Comment by paskal** [comment]
I believe it should be working. You should verify that the bot is added to the group and has "Post messages" permission.

I'll update [the documentation](https://remark42.com/docs/configuration/telegram/) to clarify that messages can be sent not only to channel but also to users and groups.

**Comment by Get-108** [comment]
It is strange, but doesn't work with groups for me. I have changed tg_id with group ID (found by web method), added bot with admin rights, turned on groups and turned off group privacy options, but there is no messages. Should i do something more? Thank you!

**Reply by paskal** [reply]
You need to verify you get the right id with some other method like [this](https://stackoverflow.com/a/72888985/961092), and if it still doesn't work please check what do you see in Remark42 logs.

**Reply by Get-108** [reply]
Yes, i used 2 methods from the link and it seems group id is correct.
Where can i see remark42 logs? Thank you!

**Reply by Get-108** [reply]
With channel ID works good, but with group – not.

**Reply by paskal** [reply]
Hi there, I just tested it, and you should use the ID provided by inviting `@myidbot` and then sending the `/getgroupid` command to chat. Old group IDs start with `-100<...>`; new ones are like `-4<...>` with fewer digits overall. For both old and new groups, channel ID is a negative number.

**Reply by irgendwr** [reply]
alternatively, here is a simple web tool that I wrote for finding the correct ID: https://irgendwr.github.io/TelegramAlert/message-tool

**Reply by paskal** [reply]
@Get-108 please let me know if your problem is solved documentation can be improved in any way.
